static/js/factories/posts.js

```diff
@@ -14,7 +14,7 @@ export const makePost = (isURLPost: boolean = false): Post => ({
   author: `justareddituser${String(Math.random())}`,
   text: isURLPost ? null : casual.text,
   url: isURLPost ? casual.url : null,
-  created: casual.moment,
+  created: casual.date('YYYY-MM-DDThh:mm:ss'),
```

Comment:
I think `casual.moment.format()` would have the same result here, although either way seems fine to me.